src/test/java/com/iota/iri/benchmarks/dbbenchmark/states/DbState.java

```diff
@@ -0,0 +1,74 @@
+package com.iota.iri.benchmarks.dbbenchmark.states;
+
+import com.iota.iri.TransactionTestUtils;
+import com.iota.iri.conf.BaseIotaConfig;
+import com.iota.iri.controllers.TransactionViewModel;
+import com.iota.iri.model.Transaction;
+import com.iota.iri.storage.PersistenceProvider;
+import com.iota.iri.storage.Tangle;
+import com.iota.iri.storage.rocksDB.RocksDBPersistenceProvider;
+import org.apache.commons.io.FileUtils;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.State;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+@State(Scope.Benchmark)
+public abstract class DbState {
+    private final File dbFolder = new File("db-clean");
+    private final File logFolder = new File("db-log-clean");
+
+    private Tangle tangle;
+    private List<TransactionViewModel> transactions;
+
+    @Param({"10", "100", "500", "1000", "3000"})
+    private int numTxsToTest;
+
+
+    public void setup() throws Exception {
+        System.out.println("-----------------------trial setup--------------------------------");
+        boolean mkdirs = dbFolder.mkdirs();
+        System.out.println("mkdirs success: " + mkdirs );
```

Comment:
We should probably directly fail/throw an error if this fails.
If it succeeds there's no need to log it right? (less=more)

Reply:
:+1: